services/FacilityService.php

```diff
@@ -191,7 +192,12 @@ public function update($data)
         $sql .= "     primary_business_entity=?,";
         $sql .= "     facility_npi=?,";
         $sql .= "     facility_code=?,";
-        $sql .= "     facility_taxonomy=?";
+        $sql .= "     facility_taxonomy=?,";
+        $sql .= "     mail_street=?,";
+        $sql .= "     mail_street2=?,";
+        $sql .= "     mail_city=?,";
+        $sql .= "     mail_state=?,";
+        $sql .= "     mail_zip=?";
```

Comment:
missing line for oid here

Reply:
above issue not yet addressed

Reply:
If I could make a suggestion. Since you're already adding to this list, may I recommend condensing `$sql` that starts on 171 to just a normal string. 20 lines of variable concatenation is a bad look and this is a good opportunity to fix it. Even if you dump it into 3 lines, the SQL can break over a line without causing an issue.

As an example:
```php
$sql = "UPDATE facility SET name=?, phone=?, fax=?, street=?...";
```